config.js.docker

```diff
@@ -0,0 +1,22 @@
+var settings = {
```

Comment:
Would there be any reason not to do this for the default config? I feel like looking in process variables is a solid way to go for things. @jflory7 @repkam09